tests/testthat/relocate_eq_assign/eq_assign_multiple_tokens_eq_only-out.R

```diff
@@ -1,9 +1,9 @@
-a <- b <- c <- d <- e <- f <- g <- 4
+a = b = c = d = e = f = g = 4
```

Comment:
yep this is allowed by Air. We'll have a linter to enforce a style in a repo though. Or perhaps this should be one of the few options.

Reply:
Discussed in https://github.com/posit-dev/air/issues/231#issuecomment-2668715581